orcid-core/src/main/java/org/orcid/core/oauth/service/OrcidOauth2TokenDetailServiceImpl.java

```diff
@@ -178,26 +183,7 @@ public void removeConflictsAndCreateNew(OrcidOauth2TokenDetail detail) {
         // not delete based on the authentication key
         orcidOauth2TokenDetailDao.removeByAuthenticationKeyOrTokenValueOrRefreshTokenValue(null, detail.getTokenValue(), detail.getRefreshTokenValue());
         orcidOauth2TokenDetailDao.persist(detail);
-    }
-    
-    /**
-     * Check if a member have a specific scope over a client
-     * @param clientId
-     * @param userName
-     * @param scopes
-     * @return true if the member have access to any of the specified scope on the specified user
-     * */
-    @Override
-    public boolean checkIfScopeIsAvailableForMember(String clientId, String userName, List<String> scopes) {
```

Comment:
Lets remove this function from the OrcidOauth2TokenDetailServiceImpl since we only need it in the OrcidApiAuthorizationSecurityAspect and it is actually a risky function to use.